apis/v1alpha2/gateway_types.go

```diff
@@ -733,9 +729,11 @@ const (
 	// protocol type is not supported.
 	ListenerReasonUnsupportedProtocol ListenerConditionReason = "UnsupportedProtocol"
 
-	// This reason is used with the "Detached" condition when
-	// the Listener could not be attached to the Gateway because the
-	// requested address is not supported.
+	// This reason is used with the "Detached" condition when the Listener could
+	// not be attached to the Gateway because the requested address is not
+	// supported. This reason could be used in a number of instances, including:
```

Comment:
Need a newline here to render the markdown bulleted list.

Reply:
Updated, thanks!